### DISCUSSION TITLE
Error in form designer window if project is set to x64

### DISCUSSION THREAD

**Discussion by washburnit** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#2736] [Q&A]
Hi,

I've set the project (based on MaterialDesign) to "x64", after that in the designer window ("form" preview) the following error appears after some time:

XamlObjectWriterException: Proprietà della raccolta 'Mocks.MaterialDesignThemes_Wpf_ColorZone_0_151276709'.'Behaviors' null.

(Proprietà della raccolta should be "collection property")

The error appears inside the form "rectangle" where all XAML controls should be in.
The compilation shows no errors and is being completed.
If I set the project to "x86" the form "preview" in the designer window appears correctly.
Should I set something else in the project?
Thank you, regards.

Roberto

**Comment by ElieTaillard** [comment]
Maybe that can help you https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/issues/2183
